enclave-runtime/Enclave.edl

```diff
@@ -79,17 +79,9 @@ enclave {
 			[out, size=mrenclave_size] uint8_t* mrenclave, uint32_t mrenclave_size);
 
 		public sgx_status_t perform_ra(
-			[in, size=genesis_hash_size] uint8_t* genesis_hash, uint32_t genesis_hash_size,
-			[in] uint32_t* nonce,
 			[in, size=w_url_size] uint8_t* w_url, uint32_t w_url_size,
-			[out, size=unchecked_extrinsic_size] uint8_t* unchecked_extrinsic, uint32_t unchecked_extrinsic_size
-		);
-
-		public sgx_status_t mock_register_enclave_xt(
-            [in, size=genesis_hash_size] uint8_t* genesis_hash, uint32_t genesis_hash_size,
-            [in] uint32_t* nonce,
-            [in, size=w_url_size] uint8_t* w_url, uint32_t w_url_size,
-            [out, size=unchecked_extrinsic_size] uint8_t* unchecked_extrinsic, uint32_t unchecked_extrinsic_size
+			[out, size=unchecked_extrinsic_size] uint8_t* unchecked_extrinsic, uint32_t unchecked_extrinsic_size,
+			int skip_ra
```

Comment:
Is bool not supported or is there another reason not to use bool here?

Reply:
Yes, bool seems not to be supported. At least I didn't find it anywhere in the intel docs..

Reply:
Must be too advanced 😸